packages/decdk/README.md

```diff
@@ -192,6 +192,8 @@ Outputs:
 
 If deCDK doesn't identify a resource type as a CDK resource, it will just pass it through to the resulting output. This means that any existing CloudFormation/SAM resources (such as `AWS::SQS::Queue`) can be used as-is.
 
+The decdk JSON schema will simply pass through any resources that have a type that includes `::`, so don't expect any validation of raw CloudFormation resource properties.
```

Comment:
But... but we could perform validation, couldn't we?

Reply:
We could, but we don't at this point.

Reply:
https://github.com/awslabs/aws-cdk/issues/2001